deploy/kubernetes/01-mysql.yaml

```
@@ -0,0 +1,80 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: mysql
+  namespace: covidshield
+spec:
+  ports:
+  - port: 3306
+  selector:
+    app: mysql
+  clusterIP: None
+---
+apiVersion: apps/v1 # for versions before 1.9.0 use apps/v1beta2
+kind: Deployment
+metadata:
+  name: mysql
+  namespace: covidshield
+spec:
+  selector:
+    matchLabels:
+      app: mysql
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: mysql
+    spec:
+      containers:
+      - image: mysql:5.7
```

is there a reason we can't use mysql 8? (i haven't tested this)

```suggestion
      - image: mysql:8.0
```

---

no issue with using mysql 8